helm/archive-node/templates/db-bootstrap.yaml

```diff
@@ -14,7 +14,7 @@ spec:
       - name: import-schema
         image: bitnami/postgresql
         command: ["bash", "-c"]
-        args: ["PGPASSWORD={{ .Values.postgresql.postgresqlPassword }} psql --username {{ .Values.postgresql.postgresqlUsername }} --host {{ tpl .Values.archive.postgresHost . }} --port {{ .Values.archive.ports.postgres }} --dbname {{ .Values.archive.postgresDB}} -f <(curl -Ls {{ .Values.archive.remoteSchemaFile }})"]
+        args: ["{{- range .Values.archive.remoteSchemaAuxFiles }} curl -o {{.name}} {{.url}} {{- end }} && PGPASSWORD={{ .Values.postgresql.postgresqlPassword }} psql --username {{ .Values.postgresql.postgresqlUsername }} --host {{ tpl .Values.archive.postgresHost . }} --port {{ .Values.archive.ports.postgres }} --dbname {{ .Values.archive.postgresDB}} -f <(curl -Ls {{ .Values.archive.remoteSchemaFile }})"]
```

Comment:
I don't like it all in one line, take a look at the tutorial here for how to space out your file while still producing a one-line result: https://helm.sh/docs/chart_template_guide/control_structures/#controlling-whitespace